BeatSaberModManager/Core/InstallerLogic.cs

```diff
@@ -35,7 +35,7 @@ public void Run()
                 StatusUpdate("Starting install sequence...");
 
                 ReleaseInfo bsipa = null;
-                if ((bsipa = releases.Find(release => release.name.ToLower() == "bsipa")) != null)
+                if ((bsipa = releases.Find(release => release.name.ToLower() == "BSIPA")) != null)
```

Comment:
This will always return false.

Reply:
You are correct, and I am a dumbass. I did actually test it and it *appeared* to work, but that may have been me not looking closely enough at the result. Fixed.